src/vos/vos_common.c

```diff
@@ -678,7 +679,7 @@ vos_self_nvme_fini(void)
 #define VOS_NVME_NR_TARGET	1
 
 static int
-vos_self_nvme_init(const char *vos_path, uint32_t tgt_id)
+vos_self_nvme_init(const char *vos_path, uint32_t tgt_id, bool use_sys_db)
```

Comment:
Minor: this 'use_sys_db' flag isn't used, it could be removed in follow-on patch.

Reply:
I missed your comments since there is no email notifications, i will remove this in this PR.